homeassistant/components/amberelectric/services.yaml

```diff
@@ -0,0 +1,16 @@
+get_forecasts:
+  fields:
+    site_id:
```

Comment:
```suggestion
    config_entry_id:
```

Reply:
You're using a config entry selector which returns a config entry id and not some site id.